azure-core/azure/core/configuration.py

```diff
@@ -55,9 +57,9 @@ def __init__(self, transport=None, **kwargs):
     def get_transport(self, **kwargs):
         if self.transport:
             return self.transport(configuration=self, **kwargs)
+        return None
 
-
-class ConnectionConfiguration(object):
+class ConnectionConfiguration:
```

Comment:
See above comment re `(object)`. This will apply throughout, except for anything in `aio` namespace - as this will be Python3 only anyway and doesn't need the `(object)`.